build.sbt

```diff
@@ -90,6 +91,8 @@ lazy val azureStorageQueue = alpakkaProject("azure-storage-queue", "azure.storag
 
 lazy val cassandra = alpakkaProject("cassandra", "cassandra", Dependencies.Cassandra)
 
+lazy val chronicleQueue = alpakkaProject("chronicle-queue", "chronicle.queue", Dependencies.ChronicleQueue)
```

Comment:
Please make sure you run tests using Chronicle Queue with JVM parameter `-ea` (Enable Assertions). Chronicle Queue has assertions and intolerances to thread migration. That's why we always recommend running such code on affinity-pool.

Reply:
thanks, done